File: puzzles/Tower_Of_Hanoi/Ruby/tower_of_hanoi.rb
def tower_of_hanoi(num, from, to, aux)
    if num >= 1
        tower_of_hanoi(num - 1, from, aux, to)
        puts "Move #{num} from #{from} to #{to}"
        tower_of_hanoi(num - 1, aux, to, from)
    end
end

tower_of_hanoi(5, 'A', 'B', 'C')

File: math/towers_of_Hanoi/python/tower_of_hanoi.py
def Tower(height,fromP,toP,aux):
	if(height>=1):
		Tower(height-1,fromP,aux,toP)
		move(fromP,toP)
		Tower(height-1,aux,toP,fromP)
		
def move(fromP,toP):
	print("Moving Disk from pole "+fromP+" to pole "+toP)
	
Tower(3,"1","2","3")